host/versioning_test.go

```diff
@@ -41,7 +47,35 @@ import (
 	"go.temporal.io/server/common/primitives/timestamp"
 )
 
-func (s *integrationSuite) TestBasicVersionUpdate() {
+type versioningIntegSuite struct {
+	// override suite.Suite.Assertions with require.Assertions; this means that s.NotNil(nil) will stop the test,
+	// not merely log an error
+	*require.Assertions
+	IntegrationBase
+}
+
+// This cluster use customized threshold for history config
```

Comment:
for matching config?

Reply:
Just a little leftover copy-paste. Bad Go habits already rubbing off on me.